packages/mask/src/extension/popups/pages/Wallet/components/WalletHeader/WalletAssetsValue.tsx

```diff
@@ -0,0 +1,45 @@
+import { Skeleton, Typography, type TypographyProps } from '@mui/material'
+import { memo, useMemo } from 'react'
+import { useContainer } from 'unstated-next'
+import { WalletContext } from '../../hooks/useWalletContext.js'
+import { sum } from 'lodash-es'
+import { formatCurrency } from '@masknet/web3-shared-base'
+import { FormattedCurrency } from '@masknet/shared'
+
+interface WalletAssetsValueProps extends TypographyProps {
+    account?: string
+    skeletonWidth?: string | number
```

Comment:
Recommend to use `SkeletonProps`.

Reply:
I prefer primitive values to avoid memo a props object.
And I think these two props is enough. If the component keeps growing and needs to do more customization to the Skeleton, we can extends to `SkeletonProps` then.